server/src/packages/pas-form/pas-form.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
+import { CrmService } from '../../crm/crm.service';
+import { PACKAGE_ATTRS } from '../packages.controller';
+import { PAS_FORM_ATTRS } from './pas-form.controller';
+import { PACKAGE_TYPE_OPTIONSET } from '../packages.service';
+
+@Injectable()
+export class PasFormService {
+  constructor(private readonly crmService: CrmService) {}
+
+  async find(id) {
+    const { records: [pasForm] } = await this.crmService.get(`dcp_pasforms`, `
+      $select=${PAS_FORM_ATTRS.join(',')}
+      &$filter=
+        dcp_pasformid eq ${id}
+      &$expand=
+        dcp_dcp_applicantinformation_dcp_pasform,
+        dcp_dcp_applicantrepinformation_dcp_pasform,
+        dcp_package($select=${PACKAGE_ATTRS.join(',')}),
+        dcp_dcp_projectbbl_dcp_pasform($filter=statecode eq 0),
+        dcp_dcp_projectaddress_dcp_pasform
+    `);
+
+    // CRM feature for providing the human-readable value for the related resource
+    const { '_dcp_projectname_value@OData.Community.Display.V1.FormattedValue': dcp_projectname } = pasForm;
+
+    return {
+      ...pasForm,
+
+      dcp_revisedprojectname: pasForm.dcp_revisedprojectname || dcp_projectname,
+    };
+  }
+
+  // Gets the lateset PAS Package on Project `projectId`
```

Comment:
We should change this comment to 
`// Gets the lateset PAS Form on Project `projectId``

Reply:
go for it